specification/schemas/Client.json

```diff
@@ -22,6 +22,14 @@
             "label": {
               "type": "string",
               "example": "Magento"
+            },
+            "grant_type": {
```

Comment:
I believe we didn't add this yet as we will not allow customers to create other clients than `client_credentials`. Any reason you have added it now?

Reply:
I added it as a read-only field, because I want to hide the delete button for our `implicit` grant tasks. As discussed, we will change this to the API only returning `client_credentials` clients.